lib/chat_api/conversations.ex

```diff
@@ -23,6 +23,19 @@ defmodule ChatApi.Conversations do
     |> Repo.all()
   end
 
+  @spec list_conversations_by_account_paginated(binary(), map()) :: Paginator.Page.t()
+  def list_conversations_by_account_paginated(account_id, filters \\ %{}) do
+    Conversation
+    |> where(account_id: ^account_id)
+    |> where(^filter_where(filters))
+    |> where([c], is_nil(c.archived_at))
+    |> order_by(desc: :last_activity_at, desc: :id)
```

Comment:
just curious, what is the `desc: :id` for? is it just the secondary ordering? if that's the case can we use `inserted_at` instead?

Reply:
yeah I decided to sort using the `id` field so that we'll have a more deterministic results. In the example in this [page](https://use-the-index-luke.com/sql/partial-results/fetch-next-page), the used the `sale_id`, and the paginator doc also uses it for their [example](https://github.com/duffelhq/paginator#usage)

The paginator also mentioned
> If you want to reap all the benefits of this method it is better that you create indexes on the columns you are using as cursor fields.


The ID field also already indexed, since it's a primary key. I'd still need to add an index for `last_activity_at`

Reply:
got it, makes sense!